pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -670,9 +812,57 @@ func appendLogfile(deviceNetworkStatus *types.DeviceNetworkStatus,
 	}
 }
 
+func handleAppInstanceSummaryCreate(ctxArg interface{}, key string,
+	statusArg interface{}) {
+	handleAppInstanceSummaryImpl(ctxArg, key, statusArg)
+}
+
+func handleAppInstanceSummaryModify(ctxArg interface{}, key string,
+	statusArg interface{}, oldStatusArg interface{}) {
+	handleAppInstanceSummaryImpl(ctxArg, key, statusArg)
+}
+
+func handleAppInstanceSummaryImpl(ctxArg interface{}, key string,
+	statusArg interface{}) {
+
+	summary := statusArg.(types.AppInstanceSummary)
+	log.Functionf("handleAppInstanceSummaryImpl: Starting %d Running %d Stopping %d Error %d", summary.TotalStarting, summary.TotalRunning, summary.TotalStopping, summary.TotalError)
+
+	// For now we only support this model
+	if model != "SIEMENS AG.SIMATIC IPC127E" {
```

Comment:
Ditto

Reply:
This is required IMO. Because Zedmanager always publishes summary irrespective of model. But since handling is only implemented for this particular model we return here. But this code helps to get trace of app status on other platforms so it is helpful for debugging.